internal/vds/vds.cpp

```diff
@@ -308,7 +308,8 @@ struct response metadata(
     nlohmann::json meta;
 
     meta["crs"] = metadata.crs();
-
+    meta["displayName"] = metadata.get_displayname();
```

Comment:
This is 100% my fault, but after looking at the OpenVDS docs I think we want [InputFileName](https://osdu.pages.opengroup.org/platform/domain-data-mgmt-services/seismic/open-vds/cppdoc/class/classOpenVDS_1_1KnownMetadata.html#_CPPv4N7OpenVDS13KnownMetadata30ImportInformationInputFileNameEv) instead.

Reply:
Sure! I replaced the display name with the input file name and function